src/dawn/engine_dawn.cc

```diff
@@ -109,54 +111,24 @@ Result MakeTexture(const ::dawn::Device& device,
   return Result("Dawn: Failed to allocate a framebuffer texture");
 }
 
-// Creates a host-side buffer for the framebuffer, and returns it through
-// |result_ptr|. The buffer will be used as a transfer destination and
+// Creates a host-side buffer  of |size| for the framebuffer, and returns it
```

Comment:
Still needs the units:

// ... buffer of |size| _bytes_ for the framebuffer...